internal/cmd/logs-files-metric-exporter/symlink-logfiles-watcher.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"flag"
+	"github.com/ViaQ/logerr/log"
+	"os"
+	"net/http"
+	"github.com/openshift/cluster-logging-operator/internal/pkg/symnotify"
+        "github.com/prometheus/client_golang/prometheus"
+        "github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+
+
+
+var (
+debugOn bool = true
```

Comment:
I presume we can remove this default value since the flags will set it accordingly